### ISSUE TITLE
Update to auto-populate registry if content doesn't exist

### ISSUE THREAD

**Issue by michaelestrin** [edgexfoundry/go-mod-bootstrap#14]
config-seed-like functionality at an individual service-level to enable config-seed service to be removed.

Dependent upon completion of https://github.com/edgexfoundry/go-mod-bootstrap/issues/13.

**Comment by lenny-intel** [comment]
@michaelestrin , you done any work on this? If not I can take this one.

**Comment by michaelestrin** [comment]
> you done any work on this? If not I can take this one

Placed this -- and related -- on hold.  See https://github.com/edgexfoundry/edgex-go/issues/2236#issuecomment-575250834 and https://github.com/edgexfoundry/edgex-go/issues/2236#issuecomment-575265608.  

I spoke with @jpwhitemn yesterday and understand he is making progress on a document to reach community consensus on what the end solution will be. 

@lenny-intel

**Comment by lenny-intel** [comment]
@michaelestrin , @jpwhitemn Why? We had already got consensus on self seed and removal of config-seed.

**Comment by michaelestrin** [comment]
@lenny-intel Subsequently discussed in today's AppFunc WG call.  We're waiting on a document from @jpwhitemn to stimulate community conversation and drive consensus/agreement around the "whole product" approach to bootstrapping an application.

**Comment by lenny-intel** [comment]
PR for this ADR https://github.com/edgexfoundry/edgex-docs/pull/115 has consensus and in process of TSC approval, so moving forward a PR for this which covers all the bootstrap related portions for the ADR.
